docs/tutorials/motion/plan-motion-with-arm-gripper.md

```diff
@@ -0,0 +1,431 @@
+{{< /alert >}}
+
+## Prerequisites
+
+Before starting this tutorial, make sure you have the [Viam Python SDK](https://python.viam.dev/) or the [Viam Go SDK](https://pkg.go.dev/go.viam.com/rdk/robot/client#section-readme) installed.
+
+If you are connecting to a real robotic arm during this tutorial, make sure your computer can communicate with the controller before continuing.
+
+Make sure you have mastery of the concepts outlined in the first Motion tutorial, [Access and Move a Robot Arm](/tutorials/motion/accessing-and-moving-robot-arm), before continuing.
+This tutorial picks up right where **Access and Move a Robot Arm** stops, so further examples depend on having a connected robot, client and service access, and other infrastructure in place.
+This also helps simplify and shorten the code examples presented below.
+
+For a helpful recap of what code we previously added, look at [the full code sample from the prior tutorial](/tutorials/motion/accessing-and-moving-robot-arm#full-tutorial-code).
+
+## Configure a Robot
+
+The robot configuration from the prior tutorial should be used for this tutorial.
+We will revisit that robot configuration and add new components during specific sections below.
+
+## Accessing the Motion Service
+
+Accessing the Motion service is very similar to accessing any other component or service within the Viam ecosystem.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+<!-- TODO: Add Python -->
+```python {class="line-numbers linkable-line-numbers"}
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+You must import an additional Go package to access the Motion service.
+Add `"go.viam.com/rdk/services/motion"` to your import list and then add the sample code below to your own client script.
+
+```go {class="line-numbers linkable-line-numbers"}
+motionService, err := motion.FromRobot(robot, "builtin")
+if err != nil {
+  logger.Fatal(err)
+}
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+Once the Motion service can be accessed, some familiar features become available.
+The Motion service has a method that can get the Pose of a component relative to a **reference frame**.
+In the tutorial where we interacted with an Arm component, the `EndPosition` method was what we used to determine the Pose of the end effector of `myArm`.
+The `GetPose` method provided by the Motion service serves a similar function to `EndPosition`, but allows for querying of Pose data with respect to other elements of the robot (such as another component or the robot's fixed "world" frame).
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+<!-- TODO: Add Python -->
+```python {class="line-numbers linkable-line-numbers"}
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+Note the use of `referenceframe.World` in the following code example.
+This is a constant string value in the RDK's `referenceframe` library that is maintained for user and programmer convenience.
```

Comment:
That is true - I actually think it can just be this
```suggestion
This is a constant string value in the `referenceframe` library that is maintained for user and programmer convenience.
```